### PR TITLE
Add Account Details Dialog

### PR DESCRIPTION
**Pull request by mankyKitty**

### CODE REVIEW

**Comment by mankyKitty** [comment]
Rebasing on top of the redesigned wallet and wiring up the dialog to the correct place.

File: frontend/src/Frontend/UI/Dialogs/AccountDetails.hs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      _ <- displayText "Account Name" (constDyn $ unAccountName (_account_name a)) "account-details__name"
+      -- Public key
+      _ <- displayText "Public Key" (constDyn . keyToText . _keyPair_publicKey $ _account_key a) "account-details__pubkey"
+      -- Chain id
+      _ <- displayText "Chain ID" (constDyn . T.pack . show $ _account_chainId a) "account-details__chain-id"
+      -- separator
+      el "hr" blank
+      -- Kadena Address
+      _ <- displayText "Kadena Address" dKAddr "account-details__kadena-address"
+      -- -- copy
```
Comment (alexfmpe) [inline frontend/src/Frontend/UI/Dialogs/AccountDetails.hs]: Extra `--` ?

File: frontend/src/Frontend/UI/Button.hs [hunk 2]
```diff
@@ -179,8 +179,11 @@ copyButton
   -> Behavior t Text
   -> m (Event t ())
 copyButton cfg t = do
-    onClick <- uiButtonDyn (cfg & uiButtonCfg_class %~ (<> "button_border_none")) $
-      elClass "span" "fa fa-lg fa-copy" blank
+    onClick <- uiButtonDyn (cfg & uiButtonCfg_class %~ (<> "button_border_none")) $ do
```
Comment (alexfmpe) [inline frontend/src/Frontend/UI/Button.hs]:
```suggestion
    onClick <- uiButtonDyn (cfg & uiButtonCfg_class <>~ "button_border_none") $ do
```

**Comment by mankyKitty** [comment]
Depends on #191

**Pull request merged**